source/state_representation/test/tests/robot/test_joint_state.cpp

```diff
@@ -158,6 +158,35 @@ TEST(JointStateTest, GetSetFields) {
   EXPECT_EQ(js.is_empty(), true);
 }
 
+TEST(JointStateTest, GetSetField) {
+  JointState js("test", 2);
+
+  // fields
+  js.set_position(1.1, 1);
```

Comment:
please also add a quartet of tests for setting by string name for better coverage